herokuapp/src/test/java/herokuapp/positive/LogoutPositive.java

```diff
@@ -0,0 +1,43 @@
+package herokuapp.positive;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.chrome.ChromeDriver;
+import pages.LoginPage;
+import pages.SecureArea;
+
+import java.util.concurrent.TimeUnit;
+
+public class LogoutPositive {
+
+    LoginPage loginPage;
+    SecureArea secureArea;
+    private ChromeDriver driver;
+
+    @BeforeEach
+    public void initPageObjects() {
+        System.setProperty("webdriver.chrome.driver", "drivers/chromedriver");
+        driver = new ChromeDriver();
+        loginPage = new LoginPage(driver);
+        secureArea = new SecureArea(driver);
+    }
+
+    @AfterEach
+    public void tearDown() {
+        driver.close();
+        driver.quit();
+    }
+
+    @Test
+    public void successfulLogin() {
```

Comment:
this is an end-to-end scenario, so it's better to add assertion after login, too